lib/tasks/que.rake

```diff
@@ -5,7 +5,27 @@ task que: 'que:exec'
 namespace :que do
   desc 'Start que worker'
   task exec: :environment do |_, args|
-    exec("que ./config/environment.rb que/prometheus #{args.extras.join}")
+    db_config = ActiveRecord::Base.connection_db_config.configuration_hash
```

Comment:
I agree with @akostadinov that it seems a bit ugly, but probably indeed it's better than monkey-patch... 

What would be nice though is to put this logic into some module under `lib` and call it from here. This will allow us to add some unit tests.

While simple, I think this logic is error-prone, and I'd rather have some unit tests.

Reply:
I created this: https://github.com/3scale/zync/pull/573/commits/d384f7a9d8a5a12692f180291ae181de83801876

And the tests: https://github.com/3scale/zync/pull/573/commits/6c03c2b63a9ad7c4761e8086ffd6a6869ad8c766